tools/downloader/converter.py

```diff
@@ -107,8 +108,15 @@ def main():
         try:
             mo_path = Path(os.environ['INTEL_OPENVINO_DIR']) / 'deployment_tools/model_optimizer/mo.py'
         except KeyError:
-            sys.exit('Unable to locate Model Optimizer. '
-                + 'Use --mo or run setupvars.sh/setupvars.bat from the OpenVINO toolkit.')
+            try:
+                # For OpenVINO from PYTHONPATH
+                mo_path = subprocess.run([args.python, '-c',
+                                          'import mo,sys; sys.stdout.write(mo.__file__)'],
+                                         check=True, stdout=subprocess.PIPE, encoding='utf-8').stdout
+                mo_path = (Path(mo_path).parent / '__main__.py').resolve(strict=True)
```

Comment:
That's not going to work correctly, because some MO arguments used for converting models refer to files inside the MO directory, and in order for them to be located correctly, `mo_path` must point to the `mo.py` script.

Reply:
Basic conversion worked for me. Do you know which arguments may be affected?

Reply:
Any model that uses `$mo_dir`.

Reply:
@dkurt There are another issue with mo installation via proposed command. not all extensions are installed. For example `extensions/front/tf/ssd_v2_support.json`

Reply:
Issue with extensions  installation has been addressed in https://github.com/openvinotoolkit/openvino/pull/5302 
@dkurt can you please finalize changes required to make converter work after installation of openvino-dev python package with model optimizer inside?